### PR TITLE
Add config value for ingress path

### PR DESCRIPTION
**Pull request by AlaaAttya**
Add config value for ingress path

by setting `ingress.path` you can specify the default ingress path for longhorn

### CODE REVIEW

File: chart/values.yaml [hunk 1]
```diff
@@ -189,6 +189,10 @@ ingress:
   ## If TLS is set to true, you must declare what secret will store the key/certificate for TLS
   tlsSecret: longhorn.local-tls
 
+  ## If ingress is enabled you can set the default ingress path
+  ## then you can access the UI by using the followig full path {{host}}+{{path}}
```
Comment (jenting) [inline chart/values.yaml:193]:
```suggestion
  ## then you can access the UI by using the following full path {{host}}+{{path}}
```

**Review by shuo-wu — COMMENTED** [review]
And we can add a subquestion for this field in `question.yml` so that users can config it via Rancher UI. Here is the example: https://github.com/longhorn/longhorn/blob/e98b0992b32cce7a3bbd0bb0ac13dc6064562661/chart/questions.yml#L568-L573

**Comment by jenting** [comment]
@AlaaAttya are you willing to address the PR comment?

**Comment by jenting** [comment]
Thank you @AlaaAttya 
If possible, would you like to address this comment https://github.com/longhorn/longhorn/pull/3097#pullrequestreview-772330624?

If not, we could file another PR to address it, thank you.

**Comment by AlaaAttya** [comment]
@jenting just update the PR. sorry for the late reply

File: chart/questions.yaml [hunk 2]
```diff
@@ -593,6 +593,12 @@ WARNING:
     type: hostname
     required: true
     label: Layer 7 Load Balancer Hostname
+  - variable: ingress.path
+      default: "/"
+      description: "If ingress is enabled you can use this variable to set the default ingress path"
+      type: string
+      required: true
+      label: ingress path
```
Comment (shuo-wu) [inline chart/questions.yaml:601]: NIT: 
```
      label: Ingress Path
```

**Review by shuo-wu — COMMENTED** [review]
Can you rebase then squash all modifications into a single commit? The modifications LGTM

File: chart/questions.yaml (hunk 2 above)
Comment (AlaaAttya) [reply chart/questions.yaml]: done 👍

**Comment by jenting** [comment]
@AlaaAttya 
could you please rebase then squash all modifications into a single commit?

**Comment by AlaaAttya** [comment]
> @AlaaAttya could you please rebase then squash all modifications into a single commit?

just did 👍

**Review by shuo-wu — DISMISSED** [review]

**Review by jenting — DISMISSED** [review]

**Comment by AlaaAttya** [comment]
@innobead is it ready to get merged?

File: chart/questions.yaml [hunk 3]
```diff
@@ -593,6 +593,12 @@ WARNING:
     type: hostname
     required: true
     label: Layer 7 Load Balancer Hostname
+  - variable: ingress.path
+      default: "/"
+      description: "If ingress is enabled you can set the default ingress path"
+      type: string
+      required: true
+      label: Ingress Path
```
Comment (innobead) [inline chart/questions.yaml:601]:
```suggestion
  - variable: ingress.path
    default: "/"
    description: "If ingress is enabled you can set the default ingress path"
    type: string
    required: true
    label: Ingress Path
```

**Review by innobead — CHANGES_REQUESTED** [review]
The formating of questions.yaml is not right, except that. LGTM.

**Comment by jenting** [comment]
> The formating of questions.yaml is not right, except that. LGTM.

I help update the format, this is ready to review.

**Review by shuo-wu — APPROVED** [review]

**Review by innobead — APPROVED** [review]

**Pull request merged**